### ISSUE TITLE
use of the nick instead of the name

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#51]
```
And addition to the options so the user is able to choose between using the 
name of the users that appear in the timeline or the nick, which i think is 
more used.
```

Original issue reported on code.google.com by `mich.ba...@gmail.com` on 21 Jan 2010 at 3:02

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 24 Jan 2010 at 12:03
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Closed by 0d6ebe63a5ca revision.
```

Original comment by `tro...@gmail.com` on 5 Feb 2010 at 2:10
- Changed state: **Fixed**
